mantidimaging/gui/windows/welcome_screen/presenter.py

```diff
@@ -72,7 +75,13 @@ def show_at_start_changed(self):
         self.settings.setValue("welcome_screen/last_run_version", versions.get_conda_installed_version())
 
     def check_issues(self):
+        issues = []
         if not versions.is_conda_uptodate():
             msg, detailed = versions.conda_update_message()
-            self.view.add_issue(msg)
+            issues.append(msg)
+            LOG.info(detailed)
+        if not self.cuda_present:
+            msg, detailed = cuda_check.not_found_message()
+            issues.append(msg)
             LOG.info(detailed)
+        self.view.add_issues("\n".join(issues))
```

Comment:
This has made it so that I always see the issues box, even when it's empty

![image](https://user-images.githubusercontent.com/9135965/102506994-a5b21a80-407b-11eb-9555-a97d7b021fe7.png)

I think we need to add issues only `if len(issues) > 0`